ocropus-linegen

```diff
@@ -114,36 +122,41 @@ assert len(lines)>0
 lines = list(set(lines))
 print("got", len(lines), "unique lines")
 
+
 def rgeometry(image,eps=0.03,delta=0.3):
-    m = array([[1+eps*randn(),0.0],[eps*randn(),1.0+eps*randn()]])
+    m = np.array([[1+eps*np.random.randn(),0.0],[eps*np.random.randn(),1.0+eps*np.random.randn()]])
     w,h = image.shape
-    c = array([w/2.0,h/2])
-    d = c-dot(m,c)+array([randn()*delta,randn()*delta])
+    c = np.array([w/2.0,h/2])
+    d = c-dot(m,c)+np.array([np.random.randn()*delta,np.random.randn()*delta])
```

Comment:
`NameError: global name 'dot' is not defined` -> Thus, I guess this should be `np.dot(...)`